test/interactor_test.rb

```diff
@@ -412,6 +412,11 @@ def initialize(_user, params)
       result.b.must_equal 2
     end
 
+    it 'returns hash values passed in the payload' do
```

Comment:
But the value is not symbolized, I am not sure if that is very clear. This test is to verify that the values of the Hash passed as an argument are not changed.

For example: Currently if we have
result = Interactor::Result.new({ m: { "key" => "value" } })
result.m => { key: "value" }
and it should be
result.m => { "key" => "value" }

To be honest I am not sure if we need this additional test. We already have a test before this one that has:

```ruby
    it 'returns all the values passed in the payload' do
      result = Hanami::Interactor::Result.new(a: 1, b: 2)
      result.a.must_equal 1
      result.b.must_equal 2
    end
``` 

Given that description the current behaviour is already failing it. Our test code is just not verifying it.
Maybe we could change this test to contemplate more cases passing a more complex hash in the constructor.

What do you think?